Subject: [PATCH] readd missing define to arch/um/Makefile-i386                                                                                                       
From: Olaf Hering <olh@suse.de>                                                                                                                                      

New in 2.6.13-rc3-git4:

scripts/Makefile.build:13: /Makefile: No such file or directory
scripts/Makefile.build:64: kbuild: Makefile.build is included improperly

the define was removed, but its still required to build some targets.

Signed-off-by: Olaf Hering <olh@suse.de>

 arch/um/Makefile-i386 |    1 +
 1 files changed, 1 insertion(+)

Index: linux-2.6.13-rc3-git4/arch/um/Makefile-i386
===================================================================
--- linux-2.6.13-rc3-git4.orig/arch/um/Makefile-i386
+++ linux-2.6.13-rc3-git4/arch/um/Makefile-i386
@@ -33,6 +33,7 @@ ifneq ($(CONFIG_GPROF),y)
 ARCH_CFLAGS += -DUM_FASTCALL
 endif
 
+SYS_UTIL_DIR	:= $(ARCH_DIR)/sys-i386/util
 SYS_HEADERS	:= $(SYS_DIR)/sc.h $(SYS_DIR)/thread.h
 
 prepare: $(SYS_HEADERS)
